main.js

```diff
@@ -1 +1,23 @@
 
+
+
+
+//Display
+// function dis(value) {
+//     document.getElementById("edu").value += value;
+// }
+
+//Calculate
+// function solve(id) {
+//     let x = document.getElementById("edu").value
+//     let y = eval(x)
+//     document.getElementById("edu").value = y
+// }
+
+//Clear
+// function clr() {
+//     let x = document.getElementById("edu").value
+//     let y = eval(x)
+//     let z = y - y
+//     document.getElementById("edu").value = z
+// }//
```

Comment:
Good start on this Roz! I'm looking forward to seeing how you grow with DOM manipulation. All of this will start coming together for you, trust me! You got this!